src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/LocalHelpers.cs

```
@@ -106,11 +113,6 @@ public static string ExecuteCommand(string fileName, string arguments, ILogger l
 
                     process.WaitForExit();
                 }
```

Won't the method silently fail if the program doesn't exit with `0`?

---

We wouldn't get any output, but that would be up to the caller to deal with.. We generally haven't cared too much about the return codes here. though.